contracts/ICO/EasyMineIco.sol

```diff
@@ -14,14 +14,20 @@ contract EasyMineIco {
   }
 
   /* Maximum duration of ICO */
-  uint256 constant public MAX_DURATION_BLOCKS = 200000;
+  uint256 public icoDuration;
```

Comment:
maybe let's keep the original name (but camel case)? it's not really the duration of the ICO (because it may be finished sooner), but the maximum possible duration